cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -114,6 +114,12 @@ message CassandraNode {
     required bool seed = 9;
     optional int64 lastRepairTimestamp = 10;
     optional int64 lastCleanupTimestamp = 11;
+
+    optional NodeLocation location = 12;
+}
+message NodeLocation {
+    required string datacenter = 1;
+    required string rack = 2;
```

Comment:
IMO it makes sense to merge this PR after #44 (due to the config-roles) after some changes - i.e. add the `NodeLocation` to `CassandraConfigRole`.

Meaning of this PR is just to use the `GossipingPropertyFileSnitch` instead of `SimpleSnitch` for on-prem installations. Although that does not help with EC2 + GCE since these snitches take the zone + region names for DC/rack. This gets a bit more complicated if DSE comes into play - but that's a different story.